core/src/main/scala/com/evolutiongaming/kafka/flow/TopicFlow.scala

```diff
@@ -78,8 +79,12 @@ object TopicFlow {
     val acquire = new TopicFlow[F] {
 
       def apply(records: ConsRecords) = {
+
         for {
           partitions <- cache.values
+          _ <- Log[F].debug(
```

Comment:
Doesn't it duplicate what's going to be logged in `ConsumerFlow`?

Reply:
Removed duplicate logging in `ConsumerFlow`